packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -81,20 +70,38 @@ export const ComboBoxDataProviderMixin = (superClass) =>
       ];
     }
 
+    constructor() {
+      super();
+
+      /**
+       * @type {DataProviderController}
+       * @private
+       */
+      this.__dataProviderController = new DataProviderController(this, {
+        size: this.size,
+        pageSize: this.pageSize,
+        placeholder: new ComboBoxPlaceholder(),
+        isPlaceholder: (item) => item instanceof ComboBoxPlaceholder,
+        dataProvider: this.dataProvider,
```

Comment:
Seems like this line is untested (all the tests still pass when I remove it).
Maybe it's not needed as we set dataProvider in the observer?

Reply:
I removed this line.